tests/conftest.py

```diff
@@ -10,3 +14,43 @@ def pytest_addoption(parser):
         type=pathlib.Path,
         help="Root of the ISO file tree."
     )
+
+# Fixtures {{{
+
+
+@pytest.fixture(scope="module")
+def version(request, host):
```

Comment:
`short_version`?

Reply:
I just moved it, but yes let's rename it :)